woocommerce/woocommerce
# CYS - Core: Create dummy products

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





With this PR, dummy products will be added to the store during the setup.

Futhermore, I created two follow-up issues:

- #43780: we have to handle the case in case some APIs during the setup returns an error
- #43456: we have to improve the Loading Screen component.

Closes #43462.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new WooCommerce installation with this version.
2. Ensure the WooCommerce Beta Tester plugin is installed and activated (available on this monorepo).
3. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag.
4. Visit the `wp-admin/admin.php?page=wc-admin&path=/customize-store`
5. Follow the process.
6. Ensure that products have been created



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

CYS - Core: Create dummy products.

#### Comment 

</details>
